webdriver/tests/bidi/browsing_context/get_tree/frames.py

```diff
@@ -5,19 +5,26 @@
 pytestmark = pytest.mark.asyncio
 
 
+@pytest.mark.parametrize("user_context", ["default", "new"])
 async def test_multiple_frames(
     bidi_session,
-    top_context,
     test_page,
     test_page2,
     test_page_multiple_frames,
+    user_context,
+    create_user_context
 ):
+    if user_context == "new":
+        user_context = await create_user_context()
```

Comment:
Please lets try to not override the fixture to reduce confusion. Maybe it would be better to rename the fixture instead?